bitnami/wordpress/values.yaml

```diff
@@ -131,6 +131,13 @@ existingWordPressConfigurationSecret:
 ## NOTE: useful if you deploy Memcached for caching database queries or you use an external cache server
 ##
 wordpressConfigureCache: false
+## @param wordpressAutoUpdateLevel Level of auto-updates to allow. Allowed values: `major`, `minor` or `none`.
+##
+wordpressAutoUpdateLevel: none
+## @param wordpressPlugins Plugins to install and activate, comma-separated. Can also be set to `all` or `none`.
+## NOTE: If set to all, only plugins that are already installed will be activated, and if set to none, no plugins will be activated
+##
+wordpressPlugins: none
```

Comment:
I wouldn't use a "comma-separated" format, but an array instead. In addition to this, shouldn't be possible to indicate what plugins to install and what plugins to activate separately?

Reply:
Well, I guess it should be possible to allow for both arrays, comma-separated and strings (like `none` or `all`). I'll update the description.

Reply:
BTW I don't think adding complexity to the users (with respect to having the option to specify plugins to activate and install separately) is something we should be doing, at least in the first iteration of these changes. I think it is prone to cause more confusion than good.

Reply:
> Well, I guess it should be possible to allow for both arrays, comma-separated and strings (like none or all)

It seems this new approach to support both a string and an array works:

```console
$ values="$(cat << EOF
wordpressPlugins:
  - foo
  - bar
EOF
)"
$ helm template wordpress bitnami/wordpress -f <(echo "$values") -s templates/deployment.yaml
(...)
            - name: WORDPRESS_PLUGINS
              value: "foo,bar"
$ helm template wordpress bitnami/wordpress -s templates/deployment.yaml
(...)
            - name: WORDPRESS_PLUGINS
              value: "none"
```